geometry_msgs/Pose target_pose
---
geometry_msgs/Pose current_pose
bool success